UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -27,6 +32,7 @@ public class Character : MonoBehaviour
     private void Awake()
     {
         characterController = GetComponent<CharacterController>();
+        slopeLimit = characterController.slopeLimit;
```

Comment:
Why do you copy over fields? This just creates double bookkeeping, and makes it more error prone. Instead you can just make the slopeLimit variable return the limit from the controller. 

Currently if the limit on the controller is changed its not reflected here.